docs/source/userguide/creating_a_new_design.rst

```diff
@@ -0,0 +1,163 @@
+    #. A `writeable_attribute_` is one whose value can be dynamically modified via a 'put' request executed in response to an action within a pre-cursor Block's execution.
+
+Attributes whose value can be specified at design-time are denoted by a highlight below the attribute value field.
+
+Manually Setting or Modifying a Block Attributes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Block attributes are specified via the 'Block Information Panel' associated with the Block you wish to configure.
+
+To configure an attribute:
+
+    #. Select the Block you wish to configure by clicking on it within the Layout Panel.  The selected Block will be highlighted and the Block Information Panel associated with it displayed on the right-hand side of the user interface.
+    #. Find the Attribute you wish to configure in the list of available Attributes.
+    #. Edit the Attribute value field as necessary:
+
+        * If the Attribute represents a list of pre-defined options select your desired value from the drop-down list.  The Attribute value field updates to reflect the selected value.
+        * If the Attribute represents an enable/disable option select the checkbox to enable or disable the Attribute.  If the checkbox is empty the Attribute is disabled .  When enabled a tick is displayed within the checkbox.  
+        * If the Attribute requires a manually specified value select the Attribute value field by clicking within it.  Delete any pre-existing content and enter your desired value.  Press the *enter* key for the value to be submitted and recorded.  Values that have been edited but not yet submitted are denoted with a 'pencil' icon.  Upon successful submission the pencil is replaced by the default information symbol.
+
+         **NB:** No data type validation is performed on manually entered values.
+
+During the registration process of a new Attribute value a spinning icon is displayed to the left of the modified Attribute.  For more information on the process this represents see `attribute_change_lifecycle_`.
+
+Upon successful registration the icon associated with the modified Attribute reverts to an information icon.
+
+In case of registration failure a red error icon is displayed next to the modified Attribute.
+
+
+Automatically Modifying Block Attributes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+I NEED TO EXPLORE THE USE OF PUT REQUESTS HERE
+
+
+.. _attribute_change_lifecycle_:
+
+The Attribute Change Lifecycle
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Attributes values modified via a Block Information Panel are recorded in the overall `design_`.  Once the Design has been updated the Attribute value takes immediate effect, influencing any executing processes as appropriate from that point forward.
+
+The round-trip from submission of a value via the user interface to utilisation in an execution environment is a non-atomic operation and therefore subject to a stochastic time delay overhead.
```

Comment:
Simpler words would be better. Maybe:
Putting a value to an Attribute takes a non-zero amount of time, while the Put is in progress \<stuff happens\>